test isolation problem bumblebee/document

when running `bin/test -m opengever.document -m opengever.bumblebee` the following failure occurs:

```
Failure in test test_does_not_queue_bumblebee_storing_if_not_digitally_available (opengever.document.tests.test_bumblebee.TestBumblebeeIntegrationWithEnabledFeature)
Traceback (most recent call last):
  File "/Users/deif/Plone/eggs/unittest2-0.5.1-py2.7.egg/unittest2/case.py", line 340, in run
    testMethod()
  File "/Users/deif/Files/workspaces/opengever.core/opengever/document/tests/test_bumblebee.py", line 85, in test_does_not_queue_bumblebee_storing_if_not_digitally_available
    self.assertEquals(0, len(queue), 'Expected no job in the queue.')
  File "/Users/deif/Plone/eggs/unittest2-0.5.1-py2.7.egg/unittest2/case.py", line 521, in assertEqual
    assertion_func(first, second, msg=msg)
  File "/Users/deif/Plone/eggs/unittest2-0.5.1-py2.7.egg/unittest2/case.py", line 514, in _baseAssertEqual
    raise self.failureException(msg)
AssertionError: 0 != 40 : Expected no job in the queue.
```

--- a/opengever/core/testing.py
+++ b/opengever/core/testing.py
@@ -440,9 +440,9 @@ class BumblebeeLayer(PloneSandboxLayer):
     def setUpZope(self, app, configurationContext):
         super(BumblebeeLayer, self).setUpZope(app, configurationContext)
 
-        queue = BumblebeeTestTaskQueue()
+        self.queue = BumblebeeTestTaskQueue()
         sm = getSiteManager()
-        sm.registerUtility(queue, provided=ITaskQueue, name='test-queue')
+        sm.registerUtility(self.queue, provided=ITaskQueue, name='test-queue')
 
     def setUpPloneSite(self, portal):
         activate_bumblebee_feature()
@@ -465,6 +465,10 @@ class BumblebeeLayer(PloneSandboxLayer):
         os.environ['BUMBLEBEE_PUBLIC_URL'] = 'http://bumblebee'
         os.environ.pop('BUMBLEBEE_INTERNAL_URL', None)
         os.environ.pop('BUMBLEBEE_DEACTIVATE', None)
+
+    def testTearDown(self):
+        self.queue.reset()
+        super(BumblebeeLayer, self).testTearDown()
 
     defaultBases = (OPENGEVER_FUNCTIONAL_TESTING,)
 

